cms-contentful/it/com/commercetools/sunrise/cms/contentful/ContentfulCmsServiceIT.java

```diff
@@ -36,11 +35,6 @@ private static String token() {
         return getValueForEnvVar(IT_CF_TOKEN);
     }
 
-    @Before
-    public void setUp() throws Exception {
-        contentfulCmsService = ContentfulCmsService.of(spaceId(), token(), PAGE_TYPE_NAME, PAGE_TYPE_ID_FIELD_NAME);
-    }
```

Comment:
Can I do that in the refactoring task? I would not like to do all in this one as this was just about adding support new types... So TODO would be:
1) extract initialization methods in it test.
2) check and eventually fix HttpClient initilization so that it's reused.